source/blog/2020-11-11-using-paypal-with-solidus.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: Using PayPal with Solidus
+date: 2020-11-11
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/2020/09/02/solidus-paypal-commerce-platform/cover.jpg
```

Comment:
I think it's better to copy this image over here as well. If we remove or change that blog post path this one will break along.

Reply:
Yeah, good idea, thanks!